### PR TITLE
update descriptions

### PR DESCRIPTION
**Pull request by liamocn**
Related issues:
#8 
https://github.com/CONP-PCNO/conp-portal/issues/370

Purpose:
This PR updates the descriptions of the field in the dats editor to those specified in https://github.com/CONP-PCNO/conp-documentation/blob/master/CONP_DATS_fields.md

### CODE REVIEW

File: src/components/forms/ExtraPropertiesForm/ExtraPropertiesForm.jsx [hunk 1]
```diff
@@ -34,14 +57,24 @@ export default function ExtraPropertiesForm(props) {
           name='Derived From'
           tooltip='The sources this dataset is derived from'
```
Comment (cmadjar) [inline src/components/forms/ExtraPropertiesForm/ExtraPropertiesForm.jsx]:
```suggestion
          tooltip='The sources this dataset is derived from.'
```

**Review by cmadjar — CHANGES_REQUESTED** [review]
@liamocn One tiny change to match was is in https://github.com/CONP-PCNO/conp-documentation/blob/master/CONP_DATS_fields.md and the fact that all sentences end with a dot.

Question: do you have some automated way to update those based on the CONP_DATS_fields.md document? Will it be easy to sync it or does one have to do that manually every time that file changes?

**Comment by liamiepops** [comment]
@cmadjar It's not straight-forward to sync because of the format difference. The .md file is markdown (which can contain links and is not just plain text) whereas what's displayed in the editor is jsx.

**Review by cmadjar — APPROVED** [review]

**Comment by cmadjar** [comment]
@liamocn thanks for the clarification. I can see how this could end up being an issue when things are not updated automatically. Hope there won't be too many updates needed...

**Pull request merged**